lib/TWCManager/Vehicle/TeslaAPI.py

```diff
@@ -227,11 +226,12 @@ def apiRefresh(self):
             "refresh_token": self.getCarApiRefreshToken(),
         }
         req = None
+        now = time.time()
```

Comment:
Whilst this fixes the code, I'm not sure we should have been defining now - I removed it for the new auth code because it's entirely superfluous, however I'd forgotten that we still use the old refresh code for token refresh hence the problems people are having with token refresh now.

Reply:
This was meant to just make it work - although I totally agree, that it might not be the perfect place. Without assigning the current time to `now` the coded was spitting out an error message and was none functional. This was just to make it "work".

Reply:
Yep, I understand, it's a high priority fix that I'd normally push straight through but given the PR had some other changes as well, it got caught up in review.

Reply:
And I totally agree on the importance of proper reviews and I am grateful that you are taking so much effort into these. I'm really sorry for delaying the code push due to the other (most probably unnecessary changes that I have done) changes that made the review mandatory. Please apologize.

Reply:
There's never a need to apologise! You're committing great improvements and I am very appreciative, it is just intended as an explanation from my side

Reply:
Thanks very much. Would you mind approving the pull request?